openapi/nexus-internal.json

```diff
@@ -3955,6 +3947,14 @@
           "last_port"
         ]
       },
+      "ProbeExternalIpKind": {
```

Comment:
was this renamed on purpose?

Reply:
Yeah -- there's already an `ExternalIp` type at https://github.com/oxidecomputer/omicron/blob/468e699bacc2b5d0b76e4990e9edfba191049247/nexus/types/src/external_api/views.rs#L428 which is exported by the Nexus external API. Basically we have several types all called something like `ExternalIp` that are mostly but not entirely isomorphic (e.g. some of them carry around database UUIDs). We should probably revisit this at some point.